code-challenges.test.js

```diff
@@ -11,22 +11,81 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+//input: lenght of sequence
+// output: sequence of specified lenght 
+// method: we use a conditinal statement to evaluate if the number is equal to two, if so we define what the outcome would be. In our else statement we tell it to look at the last number befroer the new number we just recieved in the sequence, and tell it to push the two last alues and add them together to recieve our new value.
+
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("fibSeq", () => {
+  it("returns the last two numbers added together to create a seq", ()=> {
+    expect(fibSeq(fibLength1).toEqual([1, 1, 2, 3, 5, 8]))
+    expect(fibSeq(fibLength2).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]))
+  })
+
+})
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+const newNum = fibLength1[fibLength1.length - 1] + (fibLength1[fibLength1.length - 2] || 0);
+
+const fibSeq = (number) => {
+  if(number===2)  {
+    return [1,1]
+  }
+  else  {
+    let sequence = fibSeq (number -1)
```

Comment:
I see the idea of what you want to do, and you're close to getting there! Here, you are performing a function call, which is not what you want to assign the variable to! I recommend breaking down each step you want to take, and then finding what tools you can use to get those actions done.